src/backends/perf_pt/collect.c

```diff
@@ -99,6 +99,8 @@ struct tracer_conf {
     pid_t       target_tid;         // Thread ID to trace.
     size_t      data_bufsize;       // Data buf size (in pages).
     size_t      aux_bufsize;        // AUX buf size (in pages).
+    size_t      new_trace_bufsize;  // The intial capacity (in bytes) of
```

Comment:
How about "Initial capacity (in bytes) of a trace buffer"?

Reply:
OK, we should also probably say which kind of trace buffer. The trace *storage* buffer, not the AUX buffer.

Reply:
OK.